### PR TITLE
chore: migrate nodegit to simple-git

### PR DESCRIPTION
**Pull request by nschonni**
[Preview Tests](https://deploy-preview-592--aria-at.netlify.app)

Found that the NPM install was stalling from this dependency so swapped with simple-git https://www.npmjs.com/package/simple-git

Husky/lint-staged changes were because the hooks are currently broken, as there is no `npm test` script

### CODE REVIEW

**Comment by nschonni** [comment]
@jscholes this is probably why Dependabot isn't running right now, it is likely also timing out when it runs

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm test
```
[the remark this answers is not in the document]
Comment (nschonni) [reply .husky/pre-commit]: Done

**Comment by nschonni** [comment]
I'm curious, because I can't see it, but is my guess about Dependabot failing true? https://github.com/w3c/aria-at/network/updates

**Comment by nschonni** [comment]
I'm not sure if this is a Windows-only issue, as there are other problems with the scripts not-normalizing paths but I wouldn't spend cycles on that issue, since the generation is more of a server side issue right now.

**Comment by howard-e** [comment]
Would a `branches-ignore:` for that workflow fix this specific issue instead? At least in the meantime until #621 lands?

Would you like to submit a PR for that @nschonni?

**Comment by nschonni** [comment]
I have https://github.com/w3c/aria-at/pull/354 for the skipping the generation for Dependabot

**Comment by nschonni** [comment]
Current `time npm ci`
```
real    8m21.557s
user    0m0.045s
sys     0m0.447s
```
with this PR
```
real    0m17.152s
user    0m0.076s
sys     0m0.275s
```

**Comment by howard-e** [comment]
> Current `time npm ci`
> 
> ```
> real    8m21.557s
> user    0m0.045s
> sys     0m0.447s
> ```
> 
> with this PR
> 
> ```
> real    0m17.152s
> user    0m0.076s
> sys     0m0.275s
> ```

Looks good @nschonni. I've gone ahead and merged that PR

**Comment by howard-e** [comment]
> https://github.com/w3c/aria-at/pulls/app%2Fdependabot

Sure!

File: package.json [hunk 2]
```diff
@@ -23,16 +23,16 @@
...
+    "simple-git": "^2.48.0"
   },
   "devDependencies": {
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-prettier": "^3.4.1",
-    "husky": "^7.0.0",
-    "lint-staged": "^11.1.2",
+    "husky": "^7.0.4",
```
Comment (nschonni) [inline package.json:34]: Not sure why these got changed, but I can roll it off if you want

Comment (howard-e) [reply package.json:34]: Yep, let's remove those changes for this PR

Comment (nschonni) [reply package.json]: Done

**Comment by howard-e** [comment]
Apologies here @nschonni. Seems this might need a rebase again and we can merge it in

**Comment by nschonni** [comment]
@howard-e gave it a rebase 😄 
After this lands, the CI and lockfile can be bumped to Node 16 compat

**Comment by nschonni** [comment]
@howard-e gave it a rebase 😄 
After this lands, the CI and lockfile can be bumped to Node 16 compat

**Pull request merged**